indico/modules/users/client/js/DataExport.jsx

```diff
@@ -0,0 +1,246 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import requestURL from 'indico-url:users.api_user_data_export';
+
+import {FORM_ERROR} from 'final-form';
+import _ from 'lodash';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import ReactDOM from 'react-dom';
+import {Form as FinalForm} from 'react-final-form';
+import {Button, Form, Icon, Label, List, Message, Table} from 'semantic-ui-react';
+
+import {FinalCheckbox, FinalSubmitButton, handleSubmitError} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+import {indicoAxios} from 'indico/utils/axios';
+import {camelizeKeys} from 'indico/utils/case';
+
+import './DataExport.module.scss';
+
+function DataExportForm({onSubmit, exportOptions}) {
+  return (
+    <FinalForm
+      onSubmit={onSubmit}
+      initialValues={{}}
+      initialValuesEqual={_.isEqual}
+      validate={values => {
+        if (!_.values(values).some(v => v)) {
```

Comment:
I don't think we really need lodash for this anymore: `Object.values(values),some(v => v)`